### DISCUSSION TITLE
Can't resync one folder with id

### DISCUSSION THREAD

**Discussion by cbnidk** [Z-Hub/Z-Push#75] [Q&A]
I will resync one folder
z-push-admin.php -a resync -t e8ae6414 -u hh

I get the folder id from
z-push-admin.php -a listfolders -u hh  -d akbcjem39l1cfa2ejlh9fpc1hc

I get the error wrong type and it is a 6, 44 or 48 byte long folder id, but my is 8 long

I use backend imap, can someone help

cb

**Comment by matidau** [comment]
Are you able to paste the exact error message?
